src/v2/Components/v2/ArtworkFilter/Utils/urlBuilder.tsx

```diff
@@ -6,16 +6,29 @@ import qs from "qs"
 // Utility method to convert keys of a hash into snake case.
 export const paramsToSnakeCase = params => {
   return Object.entries(params).reduce((acc, [field, value]) => {
-    return { ...acc, [snakeCase(field)]: value }
+    let snakeCased = snakeCase(field)
```

Comment:
I rewrote this to be explicit, and also handle the `i_ds <> IDs` conversion in addition to the `i_d <> ID` one.

(sidenote- In retrospect, it would have maybe have been better for our convention for the id-named parameters in MP V2 to have been straight camelcasing without the custom capitalization.)

Reply:
Do we have tests for this file? These would make for some nice specs 😅.

Reply:
Yep! Tests were updated as a part of this PR. https://github.com/artsy/force/pull/6399/files#diff-001490aca08011b9ab5857c431f8f767